pom.xml

```diff
@@ -81,6 +81,16 @@
       <artifactId>jnr-posix-api</artifactId>
       <version>3.1.7-1</version>
     </dependency>
+    <dependency>
```

Comment:
Could you please comment briefly on why new artifacts are being added (`commons-text-api`, `snakeyaml-api`, `json`)?

Reply:
When testing, the below errors appear:
```
java.io.IOException: Failed to load: Configuration as Code Plugin (configuration-as-code 1625.v27444588cc3d)
 - Update required: commons-text API Plugin (commons-text-api 1.10.0-27.vb_fa_3896786a_7) to be updated to 1.10.0-36.vc008c8fcda_7b_ or higher
 - Update required: SnakeYAML API Plugin (snakeyaml-api 1.33-90.v80dcb_3814d35) to be updated to 1.33-95.va_b_a_e3e47b_fa_4 or higher
```

Without adding the json artifact, I can't build even before testing:
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/rahul.kaukuntla/datadog-plugin/src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogAgentClient.java:[60,15] error: package org.json does not exist
[ERROR] /Users/rahul.kaukuntla/datadog-plugin/src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogAgentClient.java:[61,15] error: package org.json does not exist
```

Reply:
How do you choose version `1625.v27444588cc3d` for the configuration as code plugin? It seems like the dependencies it declares are not compatible with the version of Jenkins that you're using in tests.
If this is the case, then the answer would be to choose the proper plugin version

Reply:
You were right, the issue was that `configuration-as-code` was versioned manually. I have removed manual versioning, and as such, I didn't need to manually version snakeyaml-api and commons-text-api.